### ISSUE TITLE
Pass-by-reference ampersand displaying as pipe character?

### ISSUE THREAD

**Issue by susanBuck** [php/doc-en#1373]
I'm looking at the manual page for the asort function: https://www.php.net/manual/en/function.asort.php

The function signature looks like this:

![image](https://user-images.githubusercontent.com/237711/152076004-2e23e0f1-64fe-4933-959d-08db0f75e90c.png)

I believe that pipe `|` character next to $array should be an ampersand `&`, indicating it's passed by reference. 

Looking at the source code, it shows it should be an ampersand:

```
<code class="parameter reference">&amp;$array</code>
```

Why is it displaying as a pipe?

Seeing this in both Chrome and Firefox.

**Comment by cmb69** [comment]
Looks good for me (Chrome):
![Screenshot 2022-02-03 114735](https://user-images.githubusercontent.com/2306138/152328307-8b2db6ca-df42-4469-8a83-c3989d177568.gif)

Has anybody else the same issue as @susanBuck?

**Comment by susanBuck** [comment]
Thanks for checking.

Seems like it's a font issue specific to my system. I had someone else test it and it displayed fine for them as well. It does work on some browsers for me (Safari, Window's Chrome & Edge) - just not Mac Chrome or Firefox.

I did some independent tests and found that it seems specific to the first font being used, Fira Mono. When I removed that font, it displayed fine.

Must be something funky going on with my install of that font:

![image](https://user-images.githubusercontent.com/237711/152377871-d6984d43-8324-43f6-ba5b-80e342e7dea0.png)

Closing this out since it's system specific, not something actually wrong in the docs.

**Comment by susanBuck** [comment]
If anyone else comes across this problem, I just disabled the Fira Mono font on my system, so now it defaults to one of the other fonts in the style list and displays fine.

**Comment by enderwiggum** [comment]
Just ran into the same issue on Windows 10.  Happens in all my browsers.  I tried to re-install Fira Mono, with no luck.  Removing Fira Mono solves it.

However, with a little fiddling I noticed this:

code, pre.info, .docs .classsynopsis, .docs .classsynopsis code {
    font: normal 0.875rem / 1.5rem var(--font-family-mono);  //makes reference to Fira Mono
    overflow-x: auto;
}

If you change the 0.875rem (font-size) to 0.87rem, the problem disappears for me.  I don't know enough about font guts to explain why, but perhaps that is enough reason to change the size slightly or not use that font.

**Comment by enderwiggum** [comment]
Okay, I slurped down an example php.net URL to my webserver:

`wget  --page-requisites --convert-links --directory-prefix=/var/www/php --no-parent --adjust-extension --restrict-file-names=nocontrol,ascii https://www.php.net/manual/en/language.references.return.php`

...and it WORKS on my local copy.  I argued with Grok a bit about it, and found that (in windows 10 again) if I set my locale to UTF-8 via the "Beta: Use Unicode UTF-8 for worldwide language support" checkbox, the problem goes away.

( that checkbox is in Control Panel > Region > Administrative > Language for non-Unicode programs. )

There is some context difference that causes my client to render that font differently between php.net and my server.  I'm out of my depth on why.  Still, I think the thing to do is not use that font.  Seems like trouble if Win10 is having trouble with it.
